code/modules/ghostroles/spawner/base.dm

```diff
@@ -43,6 +43,9 @@
 	/// A lazylist of weakrefs to mobs this spawner has spawned
 	var/list/datum/weakref/spawned_mobs
 
+	/// Password required to spawn from this spawner. For event ghostroles.
+	var/password = null
```

Comment:
The issue here is, event maps are submitted in the repo then testmerged; as this password is defined in the code itself, anyone who can see the repo (so everyone) see this, which makes it kind of the equivalent of this:

![image](https://github.com/Aurorastation/Aurora.3/assets/65877598/72e2043a-8876-4661-83ba-8c6bfab42ee2)

If ghost roles needs passwords for events, it should have some sort of UI element visible only to admins where you can input the password before starting the round, or load them from some config file that the host sets (the former is probably preferrable), or something of the likes, I think